av_metrics_tool/src/main.rs

```diff
@@ -44,34 +41,58 @@ fn main() {
                 .long("json")
                 .takes_value(false),
         )
+        .arg(
+            Arg::with_name("CSV")
+                .help("Output results as CSV")
+                .long("csv")
+                .takes_value(false),
+        )
+        .arg(
+            Arg::with_name("MARKDOWN")
+                .help("Output results as Markdown")
+                .long("markdown")
+                .takes_value(false),
+        )
         .get_matches();
+
+    if [
+        cli.is_present("JSON"),
+        cli.is_present("CSV"),
+        cli.is_present("MARKDOWN"),
+    ]
+    .iter()
+    .filter(|&&x| x)
+    .count()
+        > 1
+    {
+        return Err("Cannot generate output for several formats.");
```

Comment:
Why not?

Reply:
av-metrics-tool was generating output either in text or json form. I stuck with that.
Generating several formats at once to stdout may looks like garbage and may be difficult to pipe to other commands. One option would be to provide parameters for output file names for each target format. That sounds like another issue to me.